genie-web/src/integTest/resources/com/netflix/genie/web/apis/rest/v3/controllers/JobRestControllerIntegrationTests/non-linux-runsh.txt

```diff
@@ -1,132 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Shouldn't the V3 execution tests still be comparing against these files?

Reply:
I found no reference to these files and as you can see tests pass without them...
I assumed they got removed some time ago, tho i don't remember specifically.

Reply:
It looks like I removed it in https://github.com/Netflix/genie/commit/3cc8ee9c799ae868789771d61e2c9976e0d027dc during all those massive changes but forgot to put it back in once the dual path was supported with a flag. We probably should put it back in

```// check that the generated run file is correct
        final String runShFileName = SystemUtils.IS_OS_LINUX ? "linux-runsh.txt" : "non-linux-runsh.txt";

        final String runShFile = this.resourceLoader
            .getResource(BASE_DIR + runShFileName)
            .getFile()
            .getAbsolutePath();
        final String runFileContents = new String(Files.readAllBytes(Paths.get(runShFile)), StandardCharsets.UTF_8);

        final String jobWorkingDir = this.jobDirResource.getFile().getCanonicalPath() + FILE_DELIMITER + id;
        final String expectedRunScriptContent = this.getExpectedRunContents(runFileContents, jobWorkingDir, id);

        RestAssured
            .given(this.getRequestSpecification())
            .filter(fileResultFilter)
            .when()
            .port(this.port)
            .get(JOBS_API + "/{id}/output/{filePath}", id, "run")
            .then()
            .statusCode(Matchers.is(HttpStatus.OK.value()))
            .body(Matchers.is(expectedRunScriptContent));```

Reply:
I'll take this commit out and add this missing code back in.